portfolio/src/main/java/com/google/sps/servlets/RandomStringServlet.java

```diff
@@ -8,13 +11,24 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that responds with the current date. */
-@WebServlet("/string")
+@WebServlet("/string-nao")
 public class RandomStringServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Random string 1");
-    response.getWriter().println("Random string 2");
+
+    ArrayList<String> str = new ArrayList<String>();
+    str.add("My favorite movie is the notebook");
+    str.add("My favorite icecream is vanilla");
+    str.add("I like playing voleyball");
+
+    String json = convertToJsonUsingGson(str);
+    response.getWriter().println(json);
+  }
+
+  private String convertToJsonUsingGson(str) {
```

Comment:
When I checked out your branch locally and tried to run it, this is where I get a compile error. I would expect the signature to look more like: `(ArrayList<String> str)`. Additionally, I would expect that the variable passed in here (str) is also passed onto the gson library rather than `serverStats`.